app/packages/core/src/components/Sidebar/ViewSelection/index.tsx

```diff
@@ -41,7 +41,8 @@ export default function ViewSelection() {
   const datasetName = useRecoilValue(fos.datasetName);
   const canEditSavedViews = useRecoilValue<boolean>(fos.canEditSavedViews);
   const setIsOpen = useSetRecoilState<boolean>(viewDialogOpen);
-  const savedViewParam = useRecoilValue(fos.viewName);
+  const urlParams = new URLSearchParams(window.location.search);
+  const savedViewParam = useRecoilValue(fos.viewName) || urlParams.get("view");
```

Comment:
Hmm, this shouldn't be necessary and is defaulting to the slug and not the name. Let me try to push a tweak

Reply:
yes! will remove it - was experimenting

Reply:
@benjaminpkane did you get a chance to check this?

Reply:
Things are working now